test/get_app_on_slave.py

```diff
@@ -1,41 +0,0 @@
-"""
```

Comment:
Yes, I removed the need to determine which app is on which slave by modifiying the default profile to accept all traffic on the host's subnet. So both hosts can ping both apps